### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by erenfro** [MichaelB2018/smartthings-mqtt-bridge#3]
Since this is a port of the smartthings-mqtt-bridge written by nodejs, this project does need some level of documentation suitable, and of course expanded to include the new and potentially growing list of features to be coming. API-wise, this was literally a drop-in replacement for the nodejs one without any change to the SmartThings device-handler or smartapp. But of course the addition of the discovery for HASS here, we'll be able to actually automate HomeAssistant discovery properly and completely with further updates.

**Comment by MichaelB2018** [comment]
Agreed - didn't spend too much time on it yet as I was not sure anybody would be interested in this project too. But I guess that's a chicken/egg problem. If we have some traction on this project now, I will spend the time to document (or feel free to do so too)
